### PR TITLE
chore: Add docs CI setup

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by nschonni** [comment]
Failure is expected till #173 lands

File: .circleci/config.yml [hunk 1]
```diff
@@ -44,9 +44,32 @@ jobs:
 
       - run: npm test
 
+  docs:
```
Comment (zacanger) [inline .circleci/config.yml]: Might be worth using an "anchor" for some of these repeated blocks. https://learnxinyminutes.com/docs/yaml/ I would point to the yaml spec but it's like 200 pages

File: .circleci/config.yml [hunk 2]
```diff
@@ -44,9 +44,32 @@ jobs:
...
+    docker:
+      - image: circleci/node:latest
+      - image: circleci/postgres:latest
```
Comment (zacanger) [inline .circleci/config.yml]: Probably no need for postgres heree

Comment (nschonni) [reply .circleci/config.yml]: Thanks, bad copy/paste

**Comment by nschonni** [comment]
@chrisdickinson no problem, I rebased and added an additional suppression for details/summary since one was added since the original PR

**Comment by chrisdickinson** [comment]
Merged in 44ad820! Thanks!

**Pull request closed without merging**